docs/user/build-customization.rst

```diff
@@ -256,8 +256,23 @@ If your project requires full control of the build process,
 and :ref:`extending the build process <build-customization:extend the build process>` is not enough,
 all the commands executed during builds can be overridden using the :ref:`config-file/v2:build.commands` configuration file key.
 
-For example, if your project uses `Pelican <https://blog.getpelican.com/>`_ instead of Sphinx for its documentation,
-your project could use the following configuration file:
+As Read the Docs does not have control over the build process,
+you are responsible for running all the commands required to install requirements and build your project properly.
+Once the build process finishes, the content of the ``_readthedocs/html/`` directory will be hosted.
+
+
+Examples
+++++++++
+
+This section contains some examples that can be seen as showcase of what it's possible to do with :ref:`config-file/v2:build.commands`.
```

Comment:
Some reduction for clarity

```suggestion
This section contains some examples that showcase what is possible with :ref:`config-file/v2:build.commands`.
```

Reply:
I force myself to reduce verbosity while writing, but it seems that being Argentinian always wins here 😂